pdns/dnsdist.cc

```diff
@@ -1769,7 +1769,7 @@ pdns::stat16_t g_cacheCleaningPercentage{100};
 
 static void maintThread()
 {
-  setThreadName("dnsdist/main");
+  setThreadName("maint");
```

Comment:
This is perhaps the most interesting change. This thread is *not* the `main` thread, it's a `maint[enance]` thread!